record-page/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>iSamples JSON Page</title>
+  <link href='https://fonts.googleapis.com/css?family=Open Sans' rel='stylesheet'>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous" />
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.8.0/dist/leaflet.css"
+    integrity="sha512-hoalWLoI8r4UszCkZ5kL8vayOGVae1oxXe/2A4AO6J9+580uKHDO3JdHb7NzwwzK5xr/Fs0W40kiNHxM9vyTtQ=="
+    crossorigin="" />
+  </link>
+  <script src="https://unpkg.com/leaflet@1.8.0/dist/leaflet.js"
+    integrity="sha512-BB3hKbKWOc9Ez/TAwyWxNXeoV9c1v6FIeYiBieIWkpLjauysF18NzgR1MBNBXf8/KABdlkX68nAhlwcDFLGPCQ=="
+    crossorigin=""></script>
+  <script type="application/ld+json">{
+    "$schema": "../../iSamplesSchemaBasic0.2.json",
+    "@id": "https://data.isamples.org/digitalsample/igsn/DSR000GGA",
+    "label": "VM29-179PC_82cm",
+    "sampleidentifier": "IGSN:DSR000GGA",
+    "description": "Not Provided",
+    "hasContextCategory": [
+        "Not Provided"
+    ],
+    "hasMaterialCategory": [
+        "Sediment"
+    ],
+    "hasSpecimenCategory": [
+        "Other solid object"
+    ],
+    "informalClassification": "Not Provided",
+    "keywords": [
+        "Core Piece"
+    ],
+    "producedBy": {
+        "@id": "",
+        "label": "Coring>PistonCorer",
+        "description": "Coring>PistonCorer",
+        "hasFeatureOfInterest": "Not Provided",
+        "responsibility": [
+            "corelab repository,,Sample Owner"
+        ],
+        "resultTime": "2016-09-06",
+        "samplingSite": {
+            "description": "Not Provided",
+            "label": "Not Provided",
+            "location": {
+                "elevation": "Not Provided",
+                "latitude": null,
+                "longitude": null
+            },
+            "placeName": []
+        }
+    },
+    "registrant": "corelab repository",
+    "samplingPurpose": "",
+    "curation": {
+        "label": "Not Provided",
+        "description": "Not Provided",
+        "accessConstraints": "Not Provided",
+        "curationLocation": "Not Provided",
+        "responsibility": "Not Provided"
+    },
+    "relatedResource": []
```

Comment:
This is for the test. We could replace it when we send `html` file with `fastAPI`.